### PR TITLE
Fix hamburger menu at mobile for pages with no project bar

### PR DESCRIPTION
**Pull request by spadgett**
The hamburger menu was broken on some pages where it should show the utility nav (help and user menus).

### CODE REVIEW

File: app/styles/_vertical-nav.less [hunk 1]
```diff
@@ -8,12 +8,14 @@
...
   @media(min-width: @screen-sm-min) {
-    top: @navbar-os-header-height-desktop;
+    // Hide at wider screen widths unless there is a project.
+    display: none;
   }
   .has-project-bar & {
     top: (@navbar-os-header-height-mobile + @project-bar-height-mobile);
     @media(min-width: @screen-sm-min) {
       top: (@navbar-os-header-height-desktop + @project-bar-height-desktop);
+      display: block;
```
Comment (rhamilto) [inline app/styles/_vertical-nav.less]: Nit:  alpha

Comment (spadgett) [reply app/styles/_vertical-nav.less]: Fixed

**Comment by spadgett** [comment]
[merge][severity: blocker]

**Comment by spadgett** [comment]
> Origin Web Console Test Results: FAILURE >
> (https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_web_console/140/) (Base Commit: 73c7420) (PR Branch Commit: c5a33e4)

Flake #2016

**Comment by spadgett** [comment]
[merge][severity: blocker]

**Pull request merged**